requirements-dev.txt

```diff
@@ -1,6 +1,7 @@
 -r requirements.txt
 argh~=0.26.1
 betamax~=0.4.2
+django-guardian~=1.4.9
```

Comment:
This should be in `requirements.txt` instead of `-dev`

Reply:
Done.